apps/entry/mutations.py

```diff
@@ -146,15 +149,40 @@ class Arguments:
     @staticmethod
     @permission_checker(['entry.delete_figure'])
     def mutate(root, info, id):
+        from apps.event.models import Event
+
         try:
             instance = Figure.objects.get(id=id)
         except Entry.DoesNotExist:
             return DeleteFigure(errors=[
                 dict(field='nonFieldErrors', messages=gettext('Figure does not exist.'))
             ])
+        if instance.event.review_status == Event.EVENT_REVIEW_STATUS.APPROVED:
+            Notification.send_multiple_notifications(
+                recipients=instance.event.regional_coordinators(
+                    event=instance.event
+                ),
+                actor=info.context.user,
+                type=Notification.Type.FIGURE_DELETED_IN_APPROVED_EVENT,
+                event=instance.event,
+                # TODO: Add proper describtive text
+                text=f'''
+                Figure having id {instance.id}, start_date
+                {instance.start_date}, end_date {instance.end_date}
+                was deleted.
+                '''
```

Comment:
This will add \n. Let's use this instead.
```suggestion
                text=(
                    f'Figure having id {instance.id}, start_date'
                    f' {instance.start_date}, end_date {instance.end_date}'
                    ' was deleted.'
                    '''
                )
```

Reply:
@thenav56  We decided not to add a description, we add an entry foreign key in the notification instead.